### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by abassey2**
# Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Update the repo with Spark content

## Issues Closed or Referenced

- Closes https://github.com/retaildevcrews/aks-quickstart/issues/6
- References #issue_number (this references the issue but does not close with PR)

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-		"--privileged"
-	],
+	"name": "docker101",
+	"image": "ghcr.io/retaildevcrews/kind",
+	"runArgs": ["--init", "--privileged"],
 	"overrideCommand": false,
-	"settings": { },
-	"forwardPorts": [ ],
+
+	"remoteUser": "akdc",
```
Comment (bartr) [inline .devcontainer/devcontainer.json:7]: this should be vscode instead of akdc

File: .devcontainer/post-install.sh [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
 
+# clone repos
+git clone https://github.com/retaildevcrews/webvalidate
+pushd ./webvalidate/src/app
+sed -i 's/api/zzz/' benchmark.json
+popd
+
 # install webv
-dotnet tool install -g webvalidate
+dotnet tool install -g webvalidate
```
Comment (bartr) [inline .devcontainer/post-install.sh:10]: this is already installed, so you can remove

File: .devcontainer/post-install.sh [hunk 3]
```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 
+# clone repos
+git clone https://github.com/retaildevcrews/webvalidate
```
Comment (bartr) [inline .devcontainer/post-install.sh:4]: I think you want to clone this in the .. directory - git doesn't like repos inside of repos

Comment (abassey2) [reply .devcontainer/post-install.sh:4]: Do you want to clone the repo into the workspaces directory? I believe it is already in the root directory within the docker 101 folder. The only problem with cloning it into the workspaces directory is it makes the demo a little more difficult, but I can try to adjust it if needed.

**Review by bartr — APPROVED** [review]

**Pull request merged**